devimint/src/lib.rs

```diff
@@ -14,3 +20,15 @@ pub mod gatewayd;
 pub mod tests;
 pub mod util;
 pub mod vars;
+
+pub async fn run_test<F, FF>(f: F) -> anyhow::Result<()>
+where
+    F: FnOnce(ProcessManager) -> FF,
+    FF: Future<Output = anyhow::Result<()>>,
+{
+    let args = cli::CommonArgs::parse_from::<_, ffi::OsString>(vec![]);
```

Comment:
Actually, we could probably parse command line ATM. I did it this way because I was trying unit tests first, and then test runner args were confusing it.